### PR TITLE
Let the OSD preview be bigger

### PR DESCRIPTION
**Pull request by McGiverGim**
When we add the option to make the OSD preview bigger, we limited it to 740px. Now that we have HD options, maybe is a little small.
This PR let's it grow more. It looks ok with QHD resolutions, for example:
image

and it scales nicelly to smaller ones:
image

if someone has some big 4K or more monitor a test will be appreciated. The preview must be only a little bigger than in QHD.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by nerdCopter** [comment]
* no issues found.
* smallest i can go before columns move is 1672 pixels wide. (non-issue)
* full-screen zooms rather well. (2529 pixels wide [screen minus panels])
* 4k full-screen zooms rather well. (3809 pixels wide [screen minus panels])
* can provide screenshots of all 3 if requested.

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
@McGiverGim is this still needed?

**Comment by McGiverGim** [comment]
I think yes. It's personal opinion. I think the preview must be bigger than now. It has nothing to do with the other PR.

**Comment by haslinghuis** [comment]
:+1:  testing

**Review by haslinghuis — APPROVED** [review]
Better. Much better :)

**Pull request merged**
